samples/snippets/snippets_mute_config.py

```diff
@@ -0,0 +1,236 @@
+#!/usr/bin/env python
+#
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# [START securitycenter_create_mute_config]
+from google.cloud.securitycenter_v1 import Finding
+
+
+def create_mute_rule(parent_path: str, mute_config_id: str) -> None:
+    """
+    Creates a mute configuration under a given scope that will mute
+    all new finding which match the filter/ mute rule.
+    Existing findings will NOT BE muted.
+    Args:
+        parent_path: use any one of the following three options,
+                    organizations/{organization} or
+                    folders/{folder} or
+                    projects/{project}
+        mute_config_id: Set a unique id; max of 63 chars.
+    """
+
+    from google.cloud import securitycenter
+
+    client = securitycenter.SecurityCenterClient()
+
+    mute_config = securitycenter.MuteConfig()
+    mute_config.description = "Mute low-medium IAM grants excluding 'compute' "
+    # Set mute rule(s).
+    # To construct mute rules and for supported properties, see:
+    # https://cloud.google.com/security-command-center/docs/how-to-mute-findings#console_3
```

Comment:
This might be a better URL to reference: https://cloud.google.com/security-command-center/docs/how-to-mute-findings#create_mute_rules. I think the #console_3 anchor will easily change when we add sections to the page.

Reply:
Ah! Didn't notice that.. thanks